unittests/savanna_disaster_recovery_tests.cpp

```diff
@@ -291,4 +291,30 @@ BOOST_FIXTURE_TEST_CASE(all_nodes_shutdown_with_reversible_blocks_lost, savanna_
    }));
 } FC_LOG_AND_RETHROW()
 
+
+// --------------------------------------------------------------------------------------------
+// --------------------------------------------------------------------------------------------
```

Comment:
Thanks Greg! Appreciate this. Can you add a description here:
Test starting a node from a fork database that only has a root. Verify that node can process an incoming block.

Reply:
I had written a comment already, but we can definitely update it to whatever you like.